apps/code-examples/src/app/code-examples/core/media-query/basic/demo.component.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import {
+  SkyMediaQueryTestingController,
+  provideSkyMediaQueryTesting,
+} from '@skyux/core/testing';
+
+import { DemoComponent } from './demo.component';
+
+describe('Media query demo', () => {
+  function setupTest(): {
+    fixture: ComponentFixture<DemoComponent>;
+    mediaController: SkyMediaQueryTestingController;
+  } {
+    const fixture = TestBed.createComponent(DemoComponent);
+    const mediaController = TestBed.inject(SkyMediaQueryTestingController);
+
+    return { fixture, mediaController };
+  }
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [provideSkyMediaQueryTesting()],
+    });
+  });
+
+  it('should change the breakpoint', async () => {
+    const { fixture, mediaController } = setupTest();
+
+    fixture.detectChanges();
+
+    await mediaController.expectBreakpoint('md');
```

Comment:
The default breakpoint is `md`, but I guess that's meaningless since the width of the screen (or container) on init is what matters, here. I'll adjust the test.